Sources/web3swift/Tokens/ERC1376/Web3+ERC1376.swift

```diff
@@ -66,20 +66,15 @@ protocol IERC1376: IERC20 {
 }
 
 // FIXME: Rewrite this to CodableTransaction
```

Comment:
@yaroslavyaroslav What did you mean by this comment? To use the `CodableTransaction` for all the calls performed below?

Reply:
@janndriessen @JeneaVranceanu Nope it was hint for myself at the time of 3.0.0 baking. So all of them are outdated now, just forgot to delete them totally. So please delete them when you've met them.